pkg/apb/rhcc_registry.go

```diff
@@ -63,19 +63,73 @@ func (r RHCCRegistry) LoadSpecs() ([]*Spec, int, error) {
 func (r RHCCRegistry) imageToSpec(image *Image) (*Spec, error) {
 	r.log.Debug("RHCCRegistry::imageToSpec")
 	_spec := &Spec{}
-	// Setting ID to foo because Dao expects non blank IDs
-	_spec.Id = "foo_id"
-	var name = strings.SplitAfter(image.Name, "/")
-	_spec.Name = name[len(name)-1]
-	_spec.Image = image.Name
-	_spec.Description = image.Description
+
+	req, err := http.NewRequest("GET", r.config.Url+"/v2/"+image.Name+"manifests/latest", nil)
+	if err != nil {
+		return nil, err
+	}
+
+	resp, err := http.DefaultClient.Do(req)
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
+
+	type label struct {
```

Comment:
Ah I see, I'd be ok with it being in the struct instead of being a map key. That makes sense to me. :+1: